src/main/java/racingcar/Car.java

```diff
@@ -25,12 +42,22 @@ private boolean isMove(int generateCondition) {
         return generateCondition >= MOVE_CONDITION;
     }
 
-    public String moveCountExpression(String moveExpression) {
-        String expression = "";
-        for (int i = 0; i < moveCount; i++) {
-            expression += moveExpression;
-        }
-        return expression;
+    public boolean isEqualPosition(Car car) {
+        return position.isEqualPosition(car.position);
+    }
+
+    public String carName() {
+        return carName.toString();
+    }
+
+    @Override
+    public String toString() {
+        return carName.toString() + CAR_NAME_COLON + position.toString();
```

Comment:
```suggestion
        return carName + CAR_NAME_COLON + position;
```
직접 toString()을 호출하지 않아도 괜찮을 것 같아요. :)